src/pages/api/assistant/trip.ts

```diff
@@ -0,0 +1,59 @@
+import { parseFilterQuery } from '@atb/components/transport-mode-filter/utils';
```

Comment:
I don't think we should use utils from components as such. Trying to keep responsibilities clear, so either transport mode should be a module with components or utils should be extracted in some way.